_posts/2015-05-30-hot-planet-astrobiology.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: Global warming may be key to finding E.T.
+author: neuroamanda
+---
+
+_Exoplanets that are too hot may be a sign of alien life_
+
+A new research paper suggests that scanning for waste heat radiating off of Earth-like planets may be important in the search for extraterrestrial intelligent life.
+Like the radio signals emitted by Earth, heat may serve as an interstellar smoke signal indicating energy production and consumption.
+While this not a new idea, it may be a consideration for how we build future large telescopes.
+
+With nearly 2000 exoplanets discovered so far &mdash; of which 40-odd are potentially habitable &mdash; Earth may not be as special in its ability to harbor life as once thought.
+One signal that we have been listening for in the search for extraterrestrial intelligence is radio waves, the dominant output of our rapidly industrialized Earth civilization over the past 100 years or so.
+But even these signals are becoming silenced with the end of terrestrial broadcasting; if the same thing happened to ET, what are some other large-scale signs that could indicate the presence of intelligent life?
+The trend on this planet is towards ever greater energy consumption, especially as the economy increasingly depends on storing and manipulating information.
+The more advanced a civilization is, the argument goes, the more power it generates, and the more it will move towards harnessing the power of its solar system, perhaps culminating in a [Dyson sphere](http://en.wikipedia.org/wiki/Dyson_sphere).
```

Comment:
Mentioning Dyson spheres is a bit distracting to the reader: you're talking about planetary climate change caused by intelligent life, whereas Dyson spheres involve harnessing the power of a star by building structures around it.

While they are somewhat related, I don't think Dyson spheres are worth mentioning here. I would change the sentence at "the more power it generates", and move to the next sentence from there.